dashboard/test/factories/pd_factories.rb

```diff
@@ -97,6 +97,7 @@
     mappings {[create(:pd_regional_partner_mapping, state: "AL")]}
     cost_scholarship_information "Some **important** information about scholarships."
     additional_program_information "And some _additional_ program information."
+    apps_priority_deadline_date {(Date.current + 5.days).strftime("%Y-%m-%d")}
```

Comment:
I would make this a trait on the regional partner factory, so that you can just add `:with_priority_deadline_date` and get it on any regional partner. But this works too!